i18n/en-US.properties

```diff
@@ -20,10 +20,16 @@ be.activitySidebar.activityFeed.replyInThread = Reply in thread
 be.activitySidebar.activityFeed.showReplies = See {repliesToLoadCount, plural, one {# reply} other {# replies}}
 # Text to show when a task no longer exists
 be.activitySidebar.activityFeed.taskMissingError = This task no longer exists
-# One of the dropdown options to select in order to filter comments
-be.activitySidebarFilter.status.all = All Comments
-# One of the dropdown options to select in order to filter comments
+# One of the dropdown options to select for filtering activity list in order to filter all activity
```

Comment:
minor: To make it less wordy, you could say something like `Dropdown option for filtering all activity from activity list`